schema/component.schema.json

```diff
@@ -36,6 +36,7 @@
         "ariaQuestion": {
           "type": "string",
           "title": "ariaQuestion",
+          "description": "This will be read out by screen readers instead of reading the displayTitle, body & instruction fields when focusing on the options. To be clear and concise, ensure the text encompasses only the question associated.",
```

Comment:
display title is two words in the AAT. we don't references the json names directly, but the attribute name from the content.schema.json

Reply:
Good call out. Updated.